.github/actions/build-android/action.yml

```
@@ -0,0 +1,77 @@
+# This is coded as an action rather than a workflow so the release-tools
+# repository can load it from a dynamically-chosen commit. Cross-repository
+# workflow calls must have a Git reference hard-coded in the calling workflow,
+# but actions can be run dynamically from the runner's filesystem.
+
+name: Build and test (Android)
+description: Build and test (Android)
+
+inputs:
+  triplet:
+    description: Host triplet
+    required: true
+
+runs:
+  using: composite
+
+  steps:
+    # The steps of a composite action are not clearly divided in the GitHub
+    # UI, so use a single step with ::group:: markers instead.
+    - shell: bash
+      env:
+        triplet: ${{ inputs.triplet }}
+      # The `name` of a composite action step doesn't appear anywhere in the
+      # GitHub UI, so put it in a comment on the first line instead.
+      run: |
+        # Build and test (Android)
+        echo "::group::Configure build Python"
+        ./Android/android.py configure-build
+        echo "::endgroup::"
+
+        echo "::group::Compile build Python"
```

This is a bit silly but one of the hacks is to duplicate this in comments so that it's surfaced in the log.
```suggestion
      run: |
        # Compile build Python
        echo "::group::Compile build Python"
```

(I still maintain that a reusable workflow would be nicer, though)

---

Ah, I see you've used it already.. Just didn't really separate bits into steps.